HACKING.md

```diff
@@ -354,24 +383,39 @@ git checkout -b <branchname>
 ```
 
 Make changes to the repository and commit the changes to your
-working branch. Push the changes to your forked `snapd` repository.
+local working branch.
+
+```sh
+git commit -sa -m "<feature/subject-prefix>: <brief-description-of-the-change>
+
+<more-elaborate-description-with-reasoning-about-the-changed>"
+``` 
+
+>If it's justified, one might consider several commits before pushing 
+their work to the remote repository. 
+One should strive to have only one commit per topic, and to avoid pushing 
+many topics in the same attempt.  
```

Comment:
s/attempt/feature branch/

Reply:
@mardy, I am sorry, but I don't understand the comment. :-[] Could you please explain? :-)

Reply:
It means replace "attempt" with "feature branch"

Reply:
Is it VIM syntax?

Reply:
I think it comes from the `sed` command, but yes, it's also present in ViM :-)